homeassistant/components/elmo_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,119 @@
+"""Support for e-connect Elmo alarm panel."""
+
+import logging
+
+import homeassistant.components.alarm_control_panel as alarm
```

Comment:
Only import the classes, constants and methods that you need:
```suggestion
from homeassistant.components.alarm_control_panel import AlarmControlPanel, FORMAT_NUMBER
```

Reply:
Done.